packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page/entities-tab.tsx

```diff
@@ -62,10 +62,10 @@ export const EntitiesTab: FunctionComponent = () => {
   }, [entities, activeWorkspaceAccountId]);
 
   const createGetCellContent = useCallback(
-    (rowData: TypeEntitiesRow[]) =>
+    (_rows: TypeEntitiesRow[]) =>
```

Comment:
not a big fan of `_rows`  as we will lose the unused variable warning if we ever change this piece of code, but I understand `rows` is declared above.

Reply:
Discussion moved to a [slack thread](https://hashintel.slack.com/archives/C02TWBTT3ED/p1670525761916299) _(internal)_

Reply:
Discussion here: https://hashintel.slack.com/archives/C02TWBTT3ED/p1670525761916299